doc/tips-and-tricks-using-energyplus/src/building-geometry-shading-zone-model/air-wall-open-air-connection-between-zones.tex

```diff
@@ -1,13 +1,14 @@
 \section{Air wall, Open air connection between zones}\label{air-wall-open-air-connection-between-zones}
 
-It is extremely difficult to model the interactions between thermal zones which are connected by a large opening. If the zones are controlled to the same conditions, then there is little to be gained by making them interact, so you could neglect any connections between the zones. In fact, if this is the case, you might consider combining the spaces into a single thermal zone. If you expect the zones to have significantly different temperatures and/or humidities, then use one of the following options. If they are modeled as separate zones, EnergyPlus models only what is explicitly described in the input file, so simply leaving a void (no surfaces) between two zones will accomplish nothing - the two zones will not be connected. The main interactions which occur across the dividing line between two zones which are fully open to each other are:
+Modeling the interactions between thermal zones which are connected by a large opening requires special consideration. EnergyPlus models only what is explicitly described in the input file, so simply leaving a void (no surfaces) between two zones will accomplish nothing - the two zones will not be connected. A building surface or fenestration surface with Construction:AirBoundary may be used connect the zones. Construction:AirBoundary has options for modeling the interactions which occur across the dividing line between two zones which are fully open to each other:
```

Comment:
Revamp the Tips and Tricks section on air walls to reference Construction:AirBoundary.